bin/message_queue_daemon

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env ruby
+require './config/environment.rb'
+
+adapter = Class.new do
+  def configure_start_command(command)
+    command.option :verbose, '-v', '--verbose', 'Be verbose.'
+  end
+
+  def on_start(options, helper)
+    puts "on_start #{options.inspect}"
+  end
+
+  def configure_supervisor(supervisor)
+    options = {
+      name: 'hermes.queue',
+      :path => '**',
+      :klass => 'post.hermes_message',
+      :event => 'create',
```

Comment:
This will send off the message every time it's created, even if you don't create it through Hermes. Is that the intention? Seems dangerous. Say I copy posts between different paths and end up copying Hermes messages by accident; then they will all be sent automatically.